apps/mobile/src/features/Discover/common/LabelCategoryDApp.tsx

```
@@ -2,34 +2,34 @@ import {atoms as a, useTheme} from '@yoroi/theme'
 
 type Props = {
   category: string
 }
 export const LabelCategoryDApp = ({category}: Props) => {
-  const mappedStrings = useMappedStrings()
+  const strings = useStrings()
+  const {atoms: ta, palette: p} = useTheme()
   const text = React.useMemo(
-    () => mappedStrings(category) ?? category,
-    [mappedStrings, category],
+    () => strings.useMappedStrings()(category) ?? category,
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)